src/main/java/org/openpnp/machine/reference/ReferenceNozzle.java

```diff
@@ -1365,8 +1365,14 @@ public void findIssues(Solutions solutions) {
         super.findIssues(solutions);
         try {
             if (solutions.isTargeting(Milestone.Basics)) {
-                findActuatorIssues(solutions, getVacuumActuator(), "vacuum valve", 
-                        new CommandType[] { CommandType.ACTUATE_BOOLEAN_COMMAND });
+                if (getVacuumActuator().getValueType() == ActuatorValueType.Double) {
+                    findActuatorIssues(solutions, getVacuumActuator(), "vacuum valve", 
+                        new CommandType[] { CommandType.ACTUATE_DOUBLE_COMMAND });
+                }
+                else {
+                    findActuatorIssues(solutions, getVacuumActuator(), "vacuum valve", 
+                            new CommandType[] { CommandType.ACTUATE_BOOLEAN_COMMAND });
+                }
```

Comment:
Do you know, why these changes pop up here? They are already in the `test` branch. 

Whitespace? Line-endings?

Reply:
thanks- I needed to merge in the latest openpnp:test